test/Altinn.App.Integration.Tests/_testapps/basic/App/Program.cs

```diff
@@ -83,5 +83,6 @@ void Configure()
     // Custom middleware not included in app template
 
     app.UseAuthenticationIntrospection();
+    app.UseConnectivityDiagnostics();
```

Comment:
Nitpick: Should "Pdf" be part of the name of the classes added in this PR, so that it's easy to understand that it's related to the PDF service?

Reply:
Good point, now that you mention it... Decided to add a localtest variant as well :smile: